src/requirements/data/minors/dbme.ts

```diff
@@ -38,23 +38,23 @@ const dbmeMinorRequirements: readonly CollegeOrMajorRequirement[] = [
     source: 'https://business.cornell.edu/programs/undergraduate/minors/business-engineers/',
     checker: includesWithSingleRequirement('AEM 2210', 'HADM 2230', 'NCC 5500', 'ORIE 3150'),
     fulfilledBy: 'credits',
-    minCount: 3,
+    perSlotMinCount: [3],
   },
   {
     name: 'Finance',
     description: 'AEM 2241/5241 OR HADM 2250 OR NCC 5560',
     source: 'https://business.cornell.edu/programs/undergraduate/minors/business-engineers/',
     checker: includesWithSingleRequirement('AEM 2241', 'AEM 5241', 'HADM 2250', 'NCC 5560'),
     fulfilledBy: 'credits',
-    minCount: 3,
+    perSlotMinCount: [3],
   },
   {
     name: 'Capstone course',
     description: 'AEM 4660',
     source: 'https://business.cornell.edu/programs/undergraduate/minors/business-engineers/',
     checker: includesWithSingleRequirement('AEM 4660'),
     fulfilledBy: 'credits',
-    minCount: 1.5,
+    perSlotMinCount: [1.5],
```

Comment:
you can have half a credit??

Reply:
Not sure what's going on there, just trying to preserve old behavior

Reply:
yea, this capstone course is 1.5 credits lol